cmd/clusterctl/client/alpha/rollout_rollbacker.go

```diff
@@ -78,3 +120,11 @@ func rollbackMachineDeployment(proxy cluster.Proxy, md *clusterv1.MachineDeploym
 	md.Spec.Template = revMSTemplate
 	return patchHelper.Patch(ctx, md)
 }
+
+func validateVersionSkewPolicy(cpVersion semver.Version, msVersion semver.Version) error {
+	if cpVersion.Major != msVersion.Major || cpVersion.Minor < msVersion.Minor || cpVersion.Minor-msVersion.Minor > 2 {
+		return errors.Errorf("version skew between ControlPlane %v and MachineSet %v is not supporeted by Kubernetes version skew policy."+
+			" If you want to rollback anyway, use --force option.", cpVersion, msVersion)
```

Comment:
Wouldn't the validation webhook trigger an error in this case?

Reply:
Do you mean [this](https://github.com/hiromu-a5a/cluster-api/blob/c1c328f9edc610ee519f7ea3c8e0318acdc16960/internal/controllers/machineset/machineset_preflight.go#L173-L178) ?
Strangely, I can skip this validation when I run `clusterctl alpha rollout undo`. Even I tried to rollback by modifying MS directly, it still works. Maybe I missed something.